draft-dss-star.md

```diff
@@ -171,17 +178,50 @@ A threshold secret sharing scheme with the following important properties:
 - Authenticity: Combining at least k = REPORT_THRESHOLD shares will only succeed if all
   shares correspond to the same underlying secret. Otherwise, it fails.
 
-A threshold secret sharing scheme with these properties has the following syntax:
+A threshold secret sharing scheme with these properties has the following API syntax:
 
-- Share(k, n, msg, rand, tag): Produce a k-out-of-n share of the secret `msg` bound to
-  the auxiliary data `tag` using randomness `rand`. The values k and n are integers,
-  and `msg`, `rand`, and `tag` are byte strings.
-- Recover(share_set): Combine the secret shares in `share_set`, which is of size at
+- Share(k, n, msg, rand, tag): Produce a k-out-of-n share of the secret
```

Comment:
```suggestion
- Share(k, msg, rand): Produce a k-threshold share of the secret
```
This signature doesn't match the function description below. I also don't know what the best way to describe this "thing" is, as we don't need to describe the number of points at which to evaluate the polynomial (n). I went with "k-threshold share," but there's probably something better.

Reply:
`k-threshold` seems fine to me